code/modules/mob/living/carbon/xenomorph/xenomorph.dm

```diff
@@ -475,12 +475,17 @@ Returns TRUE when loc_weeds_type changes. Returns FALSE when it doesn’t change
 	if(!COOLDOWN_CHECK(src, xeno_resting_cooldown))
 		balloon_alert(src, "Cannot get up so soon after resting!")
 		return
+	if(!COOLDOWN_CHECK(src, xeno_unresting_cooldown))
+		balloon_alert(src, "Cannot rest so soon after getting up!")
+		return
```

Comment:
you're just checking both regardless of whether you're resting or not resting?

Reply:
There's not really a reason to check if I'm resting or not

Reply:
One is for resting, the othe is for unresting. Checking them both is by definition bad.